### PR TITLE
Add cars

### PR DESCRIPTION
**Pull request by Syncwell-257**
"Add cars" added.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -52,6 +52,24 @@
...
+    t.string "brand"
+    t.string "model"
+    t.string "engine"
+    t.string "fuel_type"
+    t.string "year"
+    t.string "license_plate"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
```
Comment (apoltieva) [inline db/schema.rb:72]: Why do we have 2 tables?

**Review by apoltieva — COMMENTED** [review]
Testing views is a bit excessive

**Comment by apoltieva** [comment]
Why is our model named CarIndex when the DB schema specifies only a car model?

**Comment by apoltieva** [comment]
And yes, rn cars are not linked to any user because there's no association. Read https://guides.rubyonrails.org/association_basics.html

**Review by apoltieva — CHANGES_REQUESTED** [review]
I also see that you have one migration but 2 tables in the schema. It's because you've deleted the migrations file, but that doesn't change the database. If you migrated something you didn't want to, you should either reverse the migration or create a new one that would reverse the changes

File: app/controllers/cars_controller.rb [hunk 2]
```diff
@@ -3,7 +3,7 @@ class CarsController < ApplicationController
 
   # GET /cars or /cars.json
   def index
-    @cars = Car.all
+    @cars = Car.where(user: current_user)
```
Comment (Snick555) [inline app/controllers/cars_controller.rb:6]:
```suggestion
    @cars = current_user.cars
```

File: app/controllers/cars_controller.rb [hunk 3]
```diff
@@ -21,7 +21,7 @@ def edit
 
   # POST /cars or /cars.json
   def create
-    @car = Car.new(car_params)
+    @car = Car.new(car_params.merge(user: current_user))
```
Comment (Snick555) [inline app/controllers/cars_controller.rb:24]:
```suggestion
    @car = current_user.cars.new(car_params)
```

**Review by apoltieva — APPROVED** [review]

**Pull request merged**